test/spec/modules/mobfoxBidAdapter_spec.js

```diff
@@ -1,162 +1,142 @@
+      body: {
+        request: {
+          clicktype: 'safari',
+          clickurl: 'http://tokyo-my.mobfox.com/exchange.click.php?h=494ef76d5b0287a8b5ac8724855cb5e0',
+          cpmPrice: 50,
+          htmlString: 'test',
+          refresh: '30',
+          scale: 'no',
+          skippreflight: 'yes',
+          type: 'textAd',
+          urltype: 'link'
+        }
+      },
+      headers: {
+        get: function (header) {
+          if (header === 'X-Pricing-CPM') {
+            return 50;
+          }
+        }
+      }
+    };
+    it('test intercept response', () => {
+      let request = adapter.spec.buildRequests(bidRequest);
+      let bidResponses = adapter.spec.interpretResponse(mockServerResponse, request);
+      expect(bidResponses.length).to.equal(1);
+      expect(bidResponses[0].ad).to.equal('test');
+      expect(bidResponses[0].cpm).to.equal(50);
+      expect(bidResponses[0].creativeId).to.equal('267d72ac3f77a3f447b32cf7ebf20673');
+      expect(bidResponses[0].requestId).to.equal('5t5t5t5');
+      expect(bidResponses[0].currency).to.equal('USD');
+      expect(bidResponses[0].height).to.equal('480');
+      expect(bidResponses[0].netRevenue).to.equal(true);
+      expect(bidResponses[0].referrer).to.equal('http://tokyo-my.mobfox.com/exchange.click.php?h=494ef76d5b0287a8b5ac8724855cb5e0');
+      expect(bidResponses[0].ttl).to.equal(360);
+      expect(bidResponses[0].width).to.equal('320');
     });
 
-    it('should add a bid response', function () {
-      let bidderRequest = {
-        bidderCode: 'mobfox',
-        bids: [
-          {
-            bidId: 'bidId1',
-            bidder: 'mobfox',
-            params: {},
-            sizes: [[300, 250]],
-            placementCode: 'test-gpt-div-1234'
-          }
-        ]
+    it('test intercept response with empty server response', () => {
+      let request = adapter.spec.buildRequests(bidRequest);
+      let serverResponse = {
+        request: {
+          error: 'cannot get response'
+        }
       };
-
-      let requestParams = utils.cloneJson(mockRequestsParams.banner);
-      bidderRequest.bids[0].params = requestParams;
-      $$PREBID_GLOBAL$$._bidsRequested.push(bidderRequest);
-      // adapter needs to be called, in order for the stub to register.
-      adapter().callBids(bidderRequest);
-
-      let bidPlacementCode1 = stubAddBidResponse.getCall(0).args[0];
-      let bidResponse1 = stubAddBidResponse.getCall(0).args[1];
-      expect(bidPlacementCode1).to.equal('test-gpt-div-1234');
-      expect(bidResponse1.getStatusCode()).to.equal(CONSTANTS.STATUS.GOOD);
-      expect(bidResponse1.bidderCode).to.equal('mobfox');
-
-      expect(bidResponse1.cpm).to.equal(1);
-      expect(bidResponse1.width).to.equal(bidderRequest.bids[0].sizes[0][0]);
-      expect(bidResponse1.height).to.equal(bidderRequest.bids[0].sizes[0][1]);
-    });
-  });
+      let bidResponses = adapter.spec.interpretResponse(serverResponse, request);
```

Comment:
should prefer `const` unless the variable is definitely being reassigned. Many of these are not.